contracts/HATPaymentSplitterFactory.sol

```diff
@@ -8,24 +8,19 @@ import "./HATPaymentSplitter.sol";
 
 contract HATPaymentSplitterFactory {
     address public immutable implementation;
-    mapping(address => uint256) public nonce;
     event HATPaymentSplitterCreated(address indexed _hatPaymentSplitter);
 
     constructor (address _implementation) {
         implementation = _implementation;
     }
 
     function createHATPaymentSplitter(address[] memory _payees, uint256[] memory _shares) external returns (address result) {
-        result = Clones.cloneDeterministic(implementation, keccak256(abi.encodePacked(msg.sender, nonce[msg.sender]++)));
+        result = Clones.cloneDeterministic(implementation, keccak256(abi.encodePacked(_payees, _shares)));
```

Comment:
but this means that you can only deploy one payment splitter for each _payees,_shares combination. This may not be a problem (it may actually be a feature), but is different from the original implementation (and not just a simplification)

Reply:
Yes but it's intended, maybe should have mentioned that